core/chains/evm/config/toml/defaults/Zircuit_Sepolia.toml

```diff
@@ -15,6 +15,10 @@ BumpMin = '100 wei'
 [GasEstimator.BlockHistory]
 BlockHistorySize = 60
 
+[GasEstimator.DAOracle]
+OracleType = 'opstack'
```

Comment:
what other `OracleType`s do we expect in the future?

Reply:
ah, answered in the docs below:
>  Currently the available oracle types are: 'opstack', 'arbitrum', and 'zksync'

Reply:
there are also others that we expect in the future, I'll share a doc with you over slack